scripts/qesap/lib/cmds.py

```diff
@@ -292,6 +292,16 @@ def ansible_command_sequence(configure_data_ansible, base_project, sequence, ver
         ansible_cmd_seq.append({'cmd': ['mkdir', '-p', junit]})
 
     ssh_share = ansible_common.copy()
+
+    # This is a strategy to share the ssh host fingerprint,
+    # it is a way to run the first connection without the need to interactivly
+    # intervent to accept and recognize the remote host.
+    # It is implemented using https://docs.ansible.com/ansible/latest/command_guide/intro_adhoc.html
+    # So used binary is ansible and not ansible-playbook
```

Comment:
Suggestion:
 `so the binary used is 'ansible' instead of 'ansible-playbook'`
Or something like that. Only the word order...